### PR TITLE
DKG verification script

### PR DESCRIPTION
**Pull request by KPrasch**
Required Reviews
1

### CODE REVIEW

File: scripts/dkg/differing_ferveo_keys.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
```
Comment (KPrasch) [inline scripts/dkg/differing_ferveo_keys.py:17]:
```suggestion

File: scripts/dkg/differing_ferveo_keys.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+from nucypher.blockchain.eth.agents import (
+    ContractAgency,
+    CoordinatorAgent,
+)
+from nucypher.blockchain.eth.registry import ContractRegistry
+from nucypher.utilities.emitters import StdoutEmitter
+from nucypher.utilities.logging import GlobalLoggerSettings
+
+# Get an interactive Python session with all the NuCypher agents loaded by running:
+#    python scripts/hooks/nucypher_agents.py [OPTIONS]
```
Comment (KPrasch) [inline scripts/dkg/differing_ferveo_keys.py:36]:
```suggestion

File: scripts/dkg/differing_ferveo_keys.py (hunk 1 above)
Comment (derekpierre) [reply scripts/dkg/differing_ferveo_keys.py]: ✔️

File: scripts/dkg/differing_ferveo_keys.py (hunk 2 above)
Comment (derekpierre) [reply scripts/dkg/differing_ferveo_keys.py]: ✔️

**Review by derekpierre — APPROVED** [review]
🎸

**Pull request merged**